EESSI-install-software.sh

```diff
@@ -273,7 +279,16 @@ unset EESSI_PROJECT_INSTALL
 unset EESSI_SITE_INSTALL
 export EESSI_CVMFS_INSTALL=1
 module unload EESSI-extend
-module load EESSI-extend/${EESSI_VERSION}-easybuild
+eessi_extend_module=EESSI-extend/${EESSI_VERSION}-easybuild
+module avail ${eessi_extend_module} 2>&1 | grep "${eessi_extend_module}"
+ec=$?
+if [ ${ec} -eq 0 ]; then
+    module load ${eessi_extend_module}
+else
+    echo "Did not find ${eessi_extend_module} module; setting EASYBUILD_INSTALLPATH and EASYBUILD_EXPERIMENTAL manually"
+    export EASYBUILD_INSTALLPATH=${EESSI_PREFIX}/software/${EESSI_OS_TYPE}/${EESSI_SOFTWARE_SUBDIR_OVERRIDE}
+    export EASYBUILD_EXPERIMENTAL=1
+fi
```

Comment:
This creates an alternative installation setup that is dependent on whether the `${eessi_extend_module}` is available or not. Rather than do that we should just have a script that ensures this module is in place (something we already have for EasyBuild itself, https://github.com/EESSI/software-layer/blob/2023.06-software.eessi.io/load_easybuild_module.sh).

I'll look into extending the existing script to also install `EESSI-extend`

Reply:
Suggestion for this in https://github.com/trz42/software-layer/pull/76